modules/cudf/src/data_frame.ts

```diff
@@ -406,14 +675,29 @@ export class DataFrame<T extends TypeMap = any> {
    *
    * @example
    * ```typescript
-   * import {DataFrame, Series, Int32, Float32}  from '@rapidsai/cudf';
+   * import {DataFrame, Series}  from '@rapidsai/cudf';
    * const df = new DataFrame({
-   *  "ser_0": Series.new({type: new Int32, data: [0, 1, 2, 3, 4, 4], nullMask: [true,
-   * false, true, true, true, true]}), "ser_1": Series.new({type: new Float32, data: [0,
-   * NaN, 2, 3, 4, 4]})
+   *  a: Series.new([0, null, 2, null, 4, 4]),
+   *  b: Series.new([0, null, 2, 3, null, 4]),
+   *  c: Series.new([null, null, null, null, null, null])
    * });
-   * df.dropNulls(0); // returns df {ser_0: [0,2,3,4,4], ser_1: [0,2,3,4,4]}
-   * df.dropNulls(1); // returns df {ser_1: [0,NaN,2,3,4,4]}
+   *
+   * // delete rows with all nulls (default thresh=1)
+   * df.dropNulls(0);
+   * // return {a: [0, 2, null, 4, 4], b: [0, 2, 3, null, 4], c: [null, null, null, null,
+   * // null]}
```

Comment:
If this can't be made into one line e.g. because a linter will fail, then I think it would be better to use more vertical space and break on more "pretty-printing" boundaries